File: lec11.py
def array_lst_ops():
    arr_lst = []

    # Set or adding elements in the ArrayList
    arr_lst.append(10)
    arr_lst.append(int(input()))
    arr_lst.append(int(input()))
    arr_lst.append(40)

    print(arr_lst)

    # To know the length of the list
    print(len(arr_lst))

    # Get/render a particular element
    print(arr_lst[2])  # index starts from 0

    # Looping on the list
    for i in range(len(arr_lst)):
        print(arr_lst[i])

    # Or
    for elem in arr_lst:
        print(elem, end="\t")

    print()

    # To remove/delete element from the list
    arr_lst.pop(2)


def main():
    array_lst_ops()


main()

File: lec11.java
import java.util.Scanner;
import java.util.ArrayList;
public class lec11 {

    public static Scanner scn = new Scanner(System.in);

    public static void arrayLstOps() {
        ArrayList<Integer> arrLst = new ArrayList<>();
        // set or adding elements in the arrayList
        arrLst.add(10);
        arrLst.add(scn.nextInt());
        arrLst.add(scn.nextInt());
        arrLst.add(40);
        System.out.println(arrLst);

        // to know the length of arrayList, default size is 15 of arrayList and it will
        // vary
        arrLst.size();

        // get or render on particular element
        arrLst.get(2); // index we have to provide in the parenthesis to get an element.

        // looping on arrayLst
        for (Integer i = 0; i < arrLst.size(); i++) {
            System.out.println(arrLst.get(i));
        }
        // or
        for (Integer elem : arrLst) {
            System.out.print(elem + "\t");
        }

        // to remove or delete element from array list
        arrLst.remove(2);
    }

    public static void main(String[] args) {
        
    }
}
